### PR TITLE
9255 add divider component implementation

### PR DESCRIPTION
**Pull request by andrei-kucherov**

### CODE REVIEW

File: packages/ui-kit/src/Panel/index.tsx [hunk 1]
```diff
@@ -22,9 +22,11 @@ export function Panel({
   onClose,
   customCloseBtn,
   classes,
+  ref,
```
Comment (propakov) [inline packages/ui-kit/src/Panel/index.tsx:25]: hm, it's not used in Panel component
it looks like you were going to attach it to some element 🤔

File: packages/ui-kit/src/Divider/style.module.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
+.divider {
```
Comment (propakov) [inline packages/ui-kit/src/Divider/style.module.css:1]: in this file prettier reformats all the tabulation on save
we need to have one convention in team, not to have conflicts

**Review by propakov — APPROVED** [review]
Looks good to me! Please check these 2 little points, maybe they need some fix 👍

File: packages/ui-kit/src/Divider/style.module.css (hunk 2 above)
Comment (andrei-kucherov) [reply packages/ui-kit/src/Divider/style.module.css:1]: Looks like we need a dedicated document where we can specify code standarts. Or at least set up linter settign to highlight tabulation issue (similar as we have in DN2). Also I will change my IDE settings to set tabulation to 2 spaces in CSS )

Comment (propakov) [reply packages/ui-kit/src/Divider/style.module.css:1]: Such little code style things like tabulation (tabWidth: 2) specified in prettier already, it should work on autosave in IDE. 
By the way in DN we run it on lintStage, if you don't run prettier on autosave. It looks like we need to add the same option in this repo.

File: packages/ui-kit/src/Panel/index.tsx (hunk 1 above)
Comment (andrei-kucherov) [reply packages/ui-kit/src/Panel/index.tsx:25]: it was a kind of quickfix: **React.DetailedHTMLProps** has **ref** prop inside that has **LegasyRef** type while **Card** element uses ForwardRef and requires ref  of type **RefObject**. These two types are not compatible, so, I decided just to get rid of ref prop in props that are passed from Panel to Card and this is why I added ref here. Will think about more robust solution for this problem

**Pull request merged**